src/App.tsx

```diff
@@ -1,37 +1,91 @@
-import { useEffect, useRef, useState } from 'react'
-import mapboxgl, { NavigationControl } from 'mapbox-gl';
-import { Map } from 'mapbox-gl';
+import { useState } from 'react'
+import {
+  Map,
+  GeolocateControl,
+  FullscreenControl,
+  NavigationControl,
+  ScaleControl,
+  Source,
+  Layer,
+  SymbolLayer
+} from 'react-map-gl';
 
-import './App.css'
+import {
+  MapIcon
+} from './components/MapImage';
 
-console.log(import.meta.env.VITE_MAPBOX_TOKEN);
+import './App.scss'
 
-mapboxgl.accessToken = import.meta.env.VITE_MAPBOX_TOKEN;
+const TOKEN = import.meta.env.VITE_MAPBOX_TOKEN;
 
 function App() {
-  const mapContainer = useRef<HTMLDivElement>(null);
   const [lng, setLng] = useState(-81.200142);
   const [lat, setLat] = useState(28.602368);
   const [zoom, setZoom] = useState(15);
 
```

Comment:
I'm getting error when try to npm run build, since you have started the project in typescript, it's sending error 

```src/App.tsx:22:15 - error TS6133: 'setLng' is declared but its value is never read.

22   const [lng, setLng] = useState(-81.200142);
                 ~~~~~~

src/App.tsx:23:15 - error TS6133: 'setLat' is declared but its value is never read.

23   const [lat, setLat] = useState(28.602368);
                 ~~~~~~

src/App.tsx:24:16 - error TS6133: 'setZoom' is declared but its value is never read.

24   const [zoom, setZoom] = useState(15);
                  ~~~~~~~

src/App.tsx:26:22 - error TS6133: 'setVisibility' is declared but its value is never read.

26   const [visibility, setVisibility] = useState({
                        ~~~~~~~~~~~~~
```

Reply:
I've added a comment to the readme providing instructions on creating the env file and setting the mapbox token.

As the mapbox token has a limited number of map loads per month associated with it, I don't think it would be a good idea to distribute this publicly.

I've also removed the `set` functions from the useState block variables to remove those errors when building.

Reply:
Sounds good!